### DISCUSSION TITLE
why python test getting fail ?

### DISCUSSION THREAD

**Discussion by fms12** [fms12/DATA-STRUCTURE-AND-ALGORITHM-IN-PY#208] [Q&A] [answered]
@devvspaces why all python test cases are failing? any reason.

**Comment by devvspaces** [comment]
hi, @fms12 , i will check them now, the last pr right?

**Reply by fms12** [reply]
Ok.

**Comment by devvspaces** [comment, marked as answer]
I just checked this PR now
PR = #206 
Github action = [here](https://github.com/fms12/DATA-STRUCTURE-AND-ALGORITHM-IN-PY/actions/runs/3272151117/jobs/5390565938)

Why python test failed is because of linting error, the coding style used by contributor has some errors
![image](https://user-images.githubusercontent.com/77179231/196624043-70527dca-00c1-40cc-8975-9c91e28b3424.png)
